testapps/test_pillowtop/tests/test_user_pillow.py

```diff
@@ -57,8 +56,7 @@ def test_kafka_user_pillow(self):
     def test_kafka_user_pillow_deletion(self):
         user = self._make_and_test_user_kafka_pillow('test-kafka-user_deletion')
         # soft delete
-        user.doc_type = '{}{}'.format(user.doc_type, DELETED_SUFFIX)
-        user.save()
+        user.retire()
```

Comment:
That's right.  This test was testing something that doesn't happen - the doc type was being given a `-deleted` suffix, but in actuality we put that suffix on the `base_doc`.  I figured this would be a more accurate test if we just used the actual "retire" function, rather than mimicking it.  The advantage of that is we don't need to try to keep the test in sync with the code - if someone changes `retire` in the future, this test will continue being realistic, when otherwise it could easily be overlooked.